python/cudf/cudf/core/column/numerical.py

```diff
@@ -218,6 +219,36 @@ def as_timedelta_column(
             ),
         )
 
+    def _decimal_dtype(self):        
```

Comment:
Seems more suitable to go to `utils/dtypes.py`, and naming should be `decimal_dtype_from_numerical`?